website/content/en/docs/kubebuilder/quickstart.md

```diff
@@ -285,8 +285,12 @@ $ make run ENABLE_WEBHOOKS=false
 
 #### Build and push the image
 
-Build the operator image and push it to a registry.
+Before building the operator image, ensure the generated Dockerfile references
+the base image you want. You can change the default "runner" image `gcr.io/distroless/static:nonroot`
+by replacing its tag with another, for example `alpine:latest`, and removing
```

Comment:
I think we should use the Ubi images here as an example. 
```suggestion
by replacing its tag with another, for example `registry.access.redhat.com/ubi8/ubi-minimal:latest`, and the replacing
```

Reply:
There's a lot of other stuff we'd need to document, ex. writing an `entrypoint` script, to get UBI images to work for the new project layout as they do with the current project layout. That's a separate issue than just documenting how to use a different base image. At the very least it's not something that should go in a quickstart guide.

On the UBI note, do we want to even mention UBI images upstream? Seems like more of a downstream thing.